src/controllers/diff.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import Ajv from 'ajv';
+import { ProblemException } from '../exceptions/problem.exception';
+import { Controller} from '../interfaces';
+
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+import { DiffService } from '../services/diff.service';
+
+
+export class DiffController implements Controller {
+    public basepath = '/diff';
+    private ajv: Ajv;
+    diffService: DiffService;
+    private async difference(req: Request, res: Response, next: NextFunction) {
+        try {
+            await this.validateDiffDocumentsPath(req.body);
+        } catch (err) {
+            return next(new ProblemException({
+                type: 'invalid-diff-documents-path',
+                title: 'Invalid diff documents path',
+                status: 400,
+                detail: err.message,
+                }));
+        }
+        const { asyncapi, other } = req.body;
+        const output = await (await this.diffService.diff(asyncapi, other)).getOutput();
+        console.log(output);
+        res.status(200).json(output);
```

Comment:
Please use appropriate fields from body after changing `openapi.yaml` (please see other comment). Also you should wrap with `try catch` the use of `diff` library and handle the error in proper way like in https://github.com/asyncapi/server-api/blob/master/src/controllers/generate.controller.ts#L49